contributing/workflow.qmd

```diff
@@ -513,10 +513,85 @@ Please experiment, add new chapters and sections; we can shuffle chapter
+
+
+### Reorganizing: creating aliases and redirects
+
+As we reorganize different chapters/pages in our Cookbook, inevitably
+we'll break URLs that someone (maybe a researcher, maybe you!) has
+bookmarked. To prevent these broken links, we can use aliases. These are
+also sometimes called redirects.
+
+The Quarto documentation is here:
+<https://quarto.org/docs/websites/website-navigation.html#redirects>
+
+What this looks like:
+
+```         
+---
+title: How do I access data stored in Earthdata Cloud?
+execute:
+  eval: false
+aliases:
+  - working-with-data-in-cloud/Direct-Cloud-Access.html
+  - working-with-data-in-cloud/Access-EULA-Data.html
+---
+```
+
+As you work and render the Cookbook, notice the feedback Quarto gives
+you. When you see yellow text that says the following, this is a nice
+flag to you to include this as an alias.
+
+```         
+WARNING: Unable to resolve link target: working-with-data-in-cloud/Direct-Cloud-Access.qmd
+```
+
+This warning is also a good indication for you to update any links
+within the Cookbook that point to this old page. In the RStudio IDE, you
+can go to Edit \> Find in Files... to identify all the files in the
+Cookbook that have that link, and then you can do a search-replace to
+update them all at once.
+
+### _quarto.yml style
+
+We use a 2-line style in our `_quarto.yml` file to define how a Cookbook page is named in the navbar (`text`) and to point to its source file (`href`) like this:
+
+```
+  	- text: "Cheatsheets & Slides"
+      href: cheatsheets/index.qmd
+```
+ 
+Note: What is defined in `text` as the navbar title will take precedence over the title that is written in the target file. This doesn't change how the title appears on the Cookbook page itself.
+
+The alternative style is 1 line: 
+
+```
+    - cheatsheets/index.qmd
+```
+
+This is perfectly functional, but we think that the more explicit 2-line style helps us all see what needs to be edited to make our contributions most visible.
+
+
+### No TOC on a chapter landing page
+
+*We're thinking through this convention*
+
+We prefer not to have a table of contents (TOC) on every chapter page. Each chapter page should have short paragraph saying what chapter includes and "look at the navbar to see what’s here". 
+
+Why? When someone contributes a new Tutorial by adding their notebook and pointing to it in the `_quarto.yml` file, it is immediately discoverable in the navbar. Having to add a link to that tutorial to a list on the [Tutorials chapter page](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/tutorials/) is an extra step that a contributor might not do so it'll be less discoverable than if it only appeared in navbar.
+
+Caveat: On the Tutorials page (2023-08-08) the brief text under each tutorial heading says what you'll find in that tutorial and this can't be conveyed in navbar headings. 
```

Comment:
Rethinking this "No TOC on a chapter landing page" section because of this Caveat, which has important info. 

What if we met in the middle and  on the actual landing page added text like "below are some specific tutorials spotlighted with additional information. Please also see the navbar on the left for a full list"? What do you think?